### ISSUE TITLE
Missing wrapLength argument in ``pygame.font.Font.size``

### ISSUE THREAD

**Issue by bilhox** [pygame-community/pygame-ce#2127]
Hello,

Recently, you have introduced ``wraplength`` for fonts when rendering with ``pygame.font.Font.render``.
That's why, we should have a ``wraplength`` parameter for ``pygame.font.Font.size`` so it gives us the right height.

Something like this :
```py
pygame.font.Font.size(text : str, wraplength : int)
```

**Comment by Matiiss** [comment]
This method should also take newline characters into account.

**Comment by PurityLake** [comment]
Might be tricky enough, especially using utf-8 encoding. I can probably take a stab at it but I'll see what I can do.

**Comment by yuiio** [comment]
> This method should also take newline characters into account.

I agree with @Matiiss . One point of the key points of `pygame.font.Font.size`  is being able to anticipate the length of a string before rendering it. The result of the size obtained and the size of the rendering should be 100% identical, even with line breaks, in order to operate correctly.
